include/SDL_version.h

```diff
@@ -83,22 +83,29 @@ typedef struct SDL_version
     (x)->patch = SDL_PATCHLEVEL;                    \
 }
 
+#if SDL_MAJOR_VERSION < 3
 /**
  *  This macro turns the version numbers into a numeric value:
  *  \verbatim
     (1,2,3) -> (1203)
     \endverbatim
  *
  *  This assumes that there will never be more than 100 patchlevels.
+ *  This macro will not be available in SDL 3.x.
  */
 #define SDL_VERSIONNUM(X, Y, Z)                     \
     ((X)*1000 + (Y)*100 + (Z))
 
 /**
  *  This is the version number macro for the current SDL version.
+ *
+ *  This macro will not be available in SDL 3.x.
+ *
+ *  Deprecated, use SDL_VERSION_ATLEAST or SDL_VERSION instead.
  */
 #define SDL_COMPILEDVERSION \
     SDL_VERSIONNUM(SDL_MAJOR_VERSION, SDL_MINOR_VERSION, SDL_PATCHLEVEL)
+#endif
```

Comment:
Can you add a comment to match the opening #if?
e.g.
```c
#endif /* SDL_MAJOR_VERSION < 3 */
```
Note that this conditional will always be false in these headers, it's just serving as a marker for future use. Do we want to call that out differently in general?